packages/odyssey-react-mui/src/labs/SideNav.tsx

```diff
@@ -0,0 +1,572 @@
+      >
+        <CollapseLeftIcon
+          onClick={onClick}
+          onKeyDown={onClick}
+          sx={{
+            height: "16px",
+            width: "16px",
+            alignSelf: "center",
+            color: odysseyDesignTokens.HueNeutral400,
+          }}
+        />
+      </Box>
+    </div>
+  );
+};
+
+const SideNavHeader = ({
+  navHeaderText,
+  isCollapsible,
+  onCollapse,
+}: Pick<
+  SideNavProps,
+  "navHeaderText" | "isCollapsible" | "onCollapse"
+>): ReactNode => {
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+  return (
+    <SideNavHeaderContainer odysseyDesignTokens={odysseyDesignTokens}>
+      <Box
+        sx={{
+          marginTop: odysseyDesignTokens.Spacing2,
+        }}
+      >
+        <Heading6>{navHeaderText}</Heading6>
+      </Box>
+      {isCollapsible && <CollapseIcon onClick={onCollapse} />}
+    </SideNavHeaderContainer>
+  );
+};
+
+const SideNavListContainer = styled.ul({
+  padding: 0,
+  listStyle: "none",
+  listStyleType: "none",
+});
+
+const SectionHeader = styled("li", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})(({ odysseyDesignTokens }: { odysseyDesignTokens: DesignTokens }) => ({
+  fontSize: odysseyDesignTokens.TypographySizeOverline,
+  fontWeight: odysseyDesignTokens.TypographyWeightHeadingBold,
+  color: odysseyDesignTokens.HueNeutral600,
+  paddingTop: odysseyDesignTokens.Spacing3,
+  paddingBottom: odysseyDesignTokens.Spacing3,
+  paddingLeft: odysseyDesignTokens.Spacing4,
+  textTransform: "uppercase",
+}));
+
+const SideNavItemLabelContainer = styled("div", {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" &&
+    prop !== "isIconVisible" &&
+    prop !== "isDisabled",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  isIconVisible: boolean;
+  isDisabled?: boolean;
+}>(({ odysseyDesignTokens, isIconVisible, isDisabled }) => ({
+  width: "100%",
+  display: "flex",
+  flexWrap: "wrap",
+  alignItems: "center",
+  fontSize: odysseyDesignTokens.TypographyScale0,
+  fontWeight: odysseyDesignTokens.TypographyWeightHeading,
+  marginLeft: isIconVisible ? odysseyDesignTokens.Spacing3 : 0,
+  "& > a": {
+    color: `${odysseyDesignTokens.TypographyColorHeading} !important`,
+    pointerEvents: isDisabled ? "none" : "auto",
+  },
+  "& > a:hover": {
+    textDecoration: "none",
+    cursor: isDisabled ? "default" : "pointer",
+  },
+  "& > a:visited": {
+    color: odysseyDesignTokens.TypographyColorHeading,
+    fontSize: odysseyDesignTokens.TypographyScale0,
+  },
+}));
+
+const SideNavListItemContainer = styled("li", {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" &&
+    prop !== "isSelected" &&
+    prop !== "isDisabled",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  isSelected?: boolean;
+  disabled?: boolean;
+  isDisabled?: boolean;
+}>(({ odysseyDesignTokens, isSelected, isDisabled }) => ({
+  display: "flex",
+  alignItems: "center",
+  minHeight: "48px",
+  opacity: isDisabled ? "0.38" : "1",
```

Comment:
Where does this `0.38` come from?

Reply:
oh sorry, I'll use the appropriate token. this was added for testing purposes to see the disabled item in the storybook.

Reply:
I was able to replace the opacity with the TypographyDisabled color and aria-disabled attrs to fix the a11y errors.